gke-usage-metering/src/query.ts

```diff
@@ -1,5 +1,5 @@
 namespace gkeUsageMetering {
-  const requestTableID = 'gke_cluster_resource_usage';
+  const requestTableID = 'gke_cluster_resource_usage_partitioned';
```

Comment:
This will break a lot of users. We should leave `gke_cluster_resource_usage` as default but allow one to overwrite it.

Reply:
Please reject this pull request. I only meant to submit the first commit. Remaining was internal only.